docker/Dockerfile

```diff
@@ -114,7 +114,7 @@ FROM python:${PYTHON_VER}-slim AS poetry
 # https://python-poetry.org/docs/master/#installing-with-the-official-installer
 # This also makes it so that Poetry will *not* be included in the "final" image since it's not installed to /usr/local/
 ARG POETRY_HOME=/opt/poetry
-ARG POETRY_INSTALLER_PARALLEL=true
+ARG POETRY_INSTALLER_PARALLEL=false
```

Comment:
I'd prefer to leave this defaulted to true for local development purposes. Can we override it in the release and prerelease workflows instead?